website/docs/features/version-and-publish.md

```diff
@@ -1,6 +1,7 @@
 ---
 id: version-and-publish
 title: Version and Publish
+type: explanation
```

Comment:
Isn't this also tutorial?

Reply:
I agree that the intro of this docs page starts as a tutorial, this was before we started looking into categorizing things much better. I'd rather convert this more to an explainer and recipe IMHO.

A tutorial is purely learning oriented providing dedicated exercises for the "learner" to go through. I don't think we have any of those and not sure it's worth the effort to create them for Lerna (given the small API surface).

Looking at it from a Explainer/Recipe perspective, I'd have a division like

**Explainer:**
- Versioning (optionally to be added: general topic on versioning, semver etc..)
- Fixed/Locked mode
- Independent mode

**Recipe**
- How to use locked mode versioning
- How to use independent mode versioning (we could also just have one merged)
- How to publish a package